collector/pom.xml

```diff
@@ -38,6 +38,11 @@
       <version>4.11</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
```

Comment:
I removed the json dependency and it still built, I don't think anything else used json other than the collector. Although in the javaagent module's pom.xml there is a line that includes "org.json.**" in something. I'm not sure what that does specifically but that might need to be changed.